### ISSUE TITLE
Doesn't import external imports

### ISSUE THREAD

**Issue by SavostinVladimir** [riok/Kreya#17]
**Describe the bug**
Doesn't find external imports

**To Reproduce**
1. Create simple proto file, that imports external dependency `github.com/mwitkow/go-proto-validators/validator.proto`
```
syntax = "proto3";

package proto;

import "github.com/mwitkow/go-proto-validators/validator.proto";

service FakeService {
  rpc FakeEndpoint (FakeRequest) returns (FakeResponse);
}

message FakeRequest {
  int64 fakeField = 1 [(validator.field) = {int_gt: 0}];
}
```
2. Click on `Project->Importers->New importer`
3. Choose type `gRPC proto files` and enter importer name
4. Click `Save`
4. See error `Protobuf file github.com/mwitkow/go-proto-validators/validator.proto is referenced, but not found`

**Expected behavior**
File successfully imported

**Environment (if possible, copy the information from the error dialog or the About menu):**
 - OS: MacOS Catalina v10.15.7
 - Kreya Version 1.2.1-stable

**Comment by CommonGuy** [comment]
Importing dependencies directly from URLs is non-standard behavior as far as I know. However, we do like the idea and will implement it

**Comment by CommonGuy** [comment]
Do you develop in Go? We investigated this issue and found that you can get this working if you add `$GOPATH/src` to the import paths (see screenshot below for where to find them). For example, if your `$GOPATH` points to `/Users/username/go`, you need to add `/Users/username/go/src` as an import path, so that Kreya can find the external proto files.

We'll add functionality to add import paths with environment variables in #18 

P.S. In your proto file, the definition for FakeResponse is missing.

![image](https://user-images.githubusercontent.com/8035999/109768818-ae85a780-7bf9-11eb-82de-83b56a2989ff.png)

**Comment by SavostinVladimir** [comment]
Thank you!)

**Comment by CommonGuy** [comment]
FYI, you can now use `$GOPATH/src` directly as the import path, since the newly released v1.6 version supports system environment variables in import paths.
